modules/update-service-create-service-cli.adoc

```diff
@@ -0,0 +1,74 @@
+[id="update-service-create-service-cli_{context}"]
+= Creating an Update Service by using the CLI
+
+You can use the {product-title} CLI to create an Update Service by using the {product-title} Update Service Operator.
+
+.Procedure
+
+To create an Update Service by using the {product-title} CLI:
+
+. Configure the Update Service target namespace:
++
+[source,terminal]
+----
+$ NAMESPACE=FIXME-default-operand-namespace
+----
++
+The namespace must match the `targetNamespaces` value from the OperatorGroup.
+
+. Configure the name of the Update Service:
++
+[source,terminal]
+----
+$ NAME=example
+----
+
+. Configure the local registry and repository for the release images:
++
+[source,terminal]
+----
+$ RELEASE_IMAGES=registry.example.com/openshift/release
+----
+
+. Configure the local pullspec for the graph-data image:
++
+[source,terminal]
+----
+$ GRAPH_DATA_IMAGE=registry.example.com/openshift/graph-data:latest
+----
+
+. Create an Update Service object:
++
```

Comment:
UpdateService requires `registry` and `repository`:

```
The UpdateService "example" is invalid:
* spec.registry: Required value
* spec.repository: Required value
```

Reply:
UpdateService now only needs [`releases`][1] for this information.  In the API docs, I picked `quay.io/openshift-release-dev/ocp-release` as the example value, but yeah, `registry.example.com/openshift` would work too.

[1]: https://github.com/openshift/cincinnati-operator/blob/da6af353bd63f36bc1477d34a6bd782489a57139/api/v1/updateservice_types.go#L19-L22

Reply:
fixed